src/block/DeadBush.php

```diff
@@ -64,4 +64,12 @@ public function getFlameEncouragement() : int{
 	public function getFlammability() : int{
 		return 100;
 	}
+
+	private function isValidSupport(Block $block) : bool{
+		return $block->isSameType(VanillaBlocks::SAND())
+			|| $block->isSameType(VanillaBlocks::PODZOL())
+			|| $block->isSameType(VanillaBlocks::MYCELIUM())
+			|| $block->isSameType(VanillaBlocks::DIRT())
+			|| $block->isSameType(VanillaBlocks::HARDENED_CLAY());
```

Comment:
isSameType should not be used in this way. VanillaBlocks creates a new object for every call.

In addition, this doesn't account for red sand or stained clay.